bin/ci-builder

```diff
@@ -181,6 +181,7 @@ case "$cmd" in
             --env NO_COLOR
             --env POLAR_SIGNALS_API_TOKEN
             --env PYPI_TOKEN
+            --env NPM_TOKEN
```

Comment:
Oh, sigh, these were once sorted alphabetically. If you wouldn't mind resorting (but not the block below), that'd be great.